modules/issue_tracker/php/my_issue_tracker.class.inc

```diff
@@ -0,0 +1,380 @@
+<?php
```

Comment:
Should these 2 issue_tracker/ files be part of this PR or left over from a rebase?

Reply:
they're leftover. I'll undo the changes